channeldb/invoices.go

```diff
@@ -314,16 +314,22 @@ type HtlcAcceptDesc struct {
 // invoice.
 type InvoiceUpdateDesc struct {
 	// State is the new state that this invoice should progress to.
-	State ContractState
```

Comment:
`NextState`?

Reply:
I think the type name `UpdateDesc` already indicates that this is about the next.